go/oasis-node/cmd/common/metrics/cpu.go

```diff
@@ -0,0 +1,79 @@
+// Package metrics implements a prometheus metrics service.
```

Comment:
I think there's no need for this package doc string as it's already set in `metrics/metrics.go` (same for other collectors)